packages/daemon/src/app/controller/job.ts

```diff
@@ -41,10 +46,21 @@ export class JobController {
   }
 
   private async runJobWithContext(job: JobModel, cwd: string, verbose: boolean, pyIndex: string) {
+    const installPlugins = async () => {
+      for (const i in constants.PLUGINS) {
+        const plugin = constants.PLUGINS[i];
+        if (!job[plugin]) {
+          continue;
+        }
+        job[plugin] = await this.pluginManager.fetch(job[plugin]);
+        await this.pluginManager.install(job[plugin], pyIndex);
+      }
+    };
     if (verbose) {
       const sse = new ServerSentEmitter(this.ctx);
-      sse.emit('job created', job);
       try {
+        await installPlugins();
```

Comment:
Shall we make pending always for installing something?

Reply:
Done.